source/sonata_report.rst

```diff
@@ -86,6 +88,85 @@ For a full compartment reports, you need to specify both ``"sections": "all"`` a
         }
     }
 
+Extension of Compartment report
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To enable fine-grained control over which specific compartments are recorded, SONATA supports an extension using a ``compartment_sets.json`` file. This allows users to define sets of explicit compartment targets (e.g., ``dend[10](0.1)``) for selected node_id.
+
+File: ``compartment_sets.json``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This JSON file defines named sets of compartment targets. Each set specifies the population and a list of `(node_id, section_name, location)` tuples.
+
+.. code-block:: json
+
+  {
+    "example_compartment_set": {
+      "population": "S1nonbarrel_neurons",
+      "compartment_set": [
+        [0, "dend", 10, 0.1],
+        [1, "dend", 4, 0.1],
+        [2, "dend", 3, 0.1],
+        [3, "dend", 6, 0.3]
+      ]
+    }
+  }
+
+*   ``population``: The name of the node population these targets belong to.
+*   ``compartment_set``: A list of list, where each list is ``[node_id, section_name, section_index, location]``.
+    *   ``node_id``: The ID of the node within the specified ``population``.
+    *   ``section_name``: The name of the  NEURON section (e.g., "soma", "dend", "apic"). This should not contain the ``section_index`` e.g. ``"dend[10]"``
+    *   ``section_index``: The number of the NEURON section index e.g. for dend[10], ``section_index`` is 10. Smallest ``section_index`` is 0 in NEURON.
+    *   ``location``: The fractional distance along the section (0.0 to 1.0).
+
+This file is typically placed alongside other simulation configuration files, for example:
```

Comment:
I don't think this needs to be included; I would say that the file is subject to the `manifest` described in the sonata spec: "The "manifest" section of the config file provides a convenient handle on setting variables that point to base paths."; people can organize their files how they want within those constraints.

Reply:
Removed the file location text paragraph.